src/generate.js

```diff
@@ -17,11 +17,15 @@ self.generate = function(options) {
 	if (!options.hasOwnProperty('uppercase')) options.uppercase = true;
 	if (!options.hasOwnProperty('excludeSimilarCharacters')) options.excludeSimilarCharacters = false;
 
+	// check if must limit the number of repeating characters
+	if (!options.hasOwnProperty('repeatCharactersLimit') || typeof options.repeatCharactersLimit !== 'number' || options.repeatCharactersLimit < 2) options.repeatCharactersLimit = undefined;
```

Comment:
I just saw this new line comment. I'll update to conform to your property checks. My thinking was that, I wouldn't want the method to "blow up" if the user had provided a non-number value for this option. Perhaps another PR would be more appropriate.